### PR TITLE
Created Dashboard UI

### PR DESCRIPTION
**Pull request by uihalagedara**

### CODE REVIEW

File: src/pages/dashboard/dashboardComponenets/chart.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+				},
+			}}
+		/>
+	);
+
+	return (
+		<div>
+			<div
+				style={{
+					padding: "10px",
```
Comment (senuravihanjayadeva) [inline src/pages/dashboard/dashboardComponenets/chart.tsx:55]: Don't use inline CSS.

File: src/pages/dashboard/dashboardComponenets/dateTime.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+const DateTime: React.FC = () => {
+	return (
+		<div>
+			<div className="dateTime d-flex">
+				<div className="col-sm-6">
+					<p>{new Date().toLocaleTimeString("en-US")}</p>
+					<p>{new Date().toLocaleDateString("en-US")}</p>
+				</div>
+				<div className="col-sm" style={{ height: "50px", borderLeft: "6px solid green" }}></div>
```
Comment (senuravihanjayadeva) [inline src/pages/dashboard/dashboardComponenets/dateTime.tsx:11]: Don't use inline CSS

File: src/pages/dashboard/dashboardComponenets/chart.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+import { Bar } from "react-chartjs-2";
+
+const Chart: React.FC = () => {
+	const pieChart = (
+		<Bar
+			data={{
+				labels: ["Pending", "Completed", "Deleted"],
+				datasets: [
+					{
+						data: [1, 25, 22],
```
Comment (senuravihanjayadeva) [inline src/pages/dashboard/dashboardComponenets/chart.tsx:11]: Get these data from JSON file. 

For Labels and Datasets

File: src/routes/index.tsx [hunk 4]
```diff
@@ -14,6 +14,8 @@ import {
 } from "../pages";
 import PrivateRoute from "./PrivateRoute";
 
+import Dashboard from "../pages/dashboard/index";
```
Comment (miyurugunarathna) [inline src/routes/index.tsx:17]: This one should be imported from pages.

File: src/pages/dashboard/dashboardComponenets/mainCard.tsx [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+const MainCard: React.FC = () => {
+	return (
+		<div className="maincard">
+			<div className="card">
+				<img className="card-img-top" src="/images/ms_club_logo_crop.png" alt="MS logo" />
+				<br></br>
+				<div className="card-body">
+					<div className="card-deck" style={{ display: "flex" }}>
```
Comment (senuravihanjayadeva) [inline src/pages/dashboard/dashboardComponenets/mainCard.tsx:10]: Cant we use Row Column instead of flex

**Review by miyurugunarathna — CHANGES_REQUESTED** [review]
Link your assigned issues.

**Review by senuravihanjayadeva — CHANGES_REQUESTED** [review]

**Review by senuravihanjayadeva — APPROVED** [review]

**Pull request merged**